pkg/pipelines/tekton/tasks.go

```diff
@@ -228,3 +237,186 @@ spec:
       emptyDir: {}
     - name: layers-dir
       emptyDir: {}
+`
+}
+
+func getS2ITask() string {
+	return `apiVersion: tekton.dev/v1
```

Comment:
At first I was worried that `Task` CRD may evolve overtime and break at some point. But on a second though it should be stable enough. It doesn't seem to leverage any exotic parts. :)

Reply:
Also I wonder if we could reduce dependency on Tekton libraries to bare minimum. That's not directly related to the PR, rather an afterthought.